tests/graphql/test-http-baseurl.graphql

```diff
@@ -1,5 +1,5 @@
 #> server-sdl
-schema @server(upstream: {baseURL: "http://abc.com"}) {
+schema @server @upstream(baseURL: "http://abc.com") {
```

Comment:
Can we not live without `@server`, is it a required directive?

Reply:
I am not quite sure about it. In the codebase and integration test, it seems required so far.